File: C++/Program/Produk.cpp
#include <iostream>
#include <string>

using namespace std;


class Produk {
    private:
    int kodeProduk;
    int stok;
    int harga;

  public:
    Produk(){


    }

    //construktor dengan parameter

    Produk(int kodeProduk, int stok, int harga) {

        this->kodeProduk= kodeProduk;
        this->stok = stok;
        this->harga = harga;
        
    }
    //getter dan setter

    
       int getKodeProduk(){
            return kodeProduk;
        }
        void setKodeProduk(int kodeProduk){
            this->kodeProduk = kodeProduk;
        }

        int getStok(){
            return stok;
        }
        void setStok(int stok){
            this->stok = stok;
        }

    
        int getHarga(){
            return harga;
        }
         void setHarga(int harga){
            this->harga = harga;
        }

    


};

File: php/Program/Produk.php
<?php
class Produk {
    private $kodeProduk;
    private $stok;
    private $harga;

    public function __construct($kodeProduk, $stok, $harga) {
        $this->kodeProduk = $kodeProduk;
        $this->stok = $stok;
        $this->harga = $harga;
    }

    //getter dan setter
    public function getKodeProduk() {
        return $this->kodeProduk;
    }
    public function setKodeProduk($kodeProduk) {
        $this->kodeProduk = $kodeProduk;
    }
    public function getStok() {
        return $this->stok;
    }
    public function setStok($stok) {
        $this->stok = $stok;
    }

    public function getHarga() {
        return $this->harga;
    }
    public function setHarga($harga) {
        $this->harga = $harga;
    }





}
